### PR TITLE
fix(TDP-5439) : IT tests failures (maintenance/2.3)

### PR DESCRIPTION
**Pull request by tlnd-jbrun**
**Link to the JIRA issue**
https://jira.talendforge.org/browse/TDP-5439

**Please check if the PR fulfills these requirements**
- [x] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] The code coverage on new code is > 75 % for backend and > 95% for frontend
- [x] The new code does not introduce new technical issues (sonar / eslint)
- [x] Functional tests have been performed
- [ ] Docker configuration files for config-std or config-cloud profiles are impacted

**Please check the browsers you've tested on**
- [ ] Chrome, Firefox, Safari, Edge, IE11
- [ ] No, that's bad, this PR should not be merged !
- [ ] No, and no need to (backend changes only)

### CODE REVIEW

File: dataprep-test-api/src/test/java/org/talend/dataprep/qa/step/PreparationStep.java [hunk 1]
```diff
@@ -75,7 +76,7 @@ public void movePreparation(String preparationName, DataTable dataTable) throws
         Map<String, String> params = dataTable.asMap(String.class, String.class);
         List<Folder> folders = folderUtil.listFolders();
         Folder originFolder = folderUtil.extractFolder(params.get(ORIGIN), folders);
```
Comment (yacarof) [inline dataprep-test-api/src/test/java/org/talend/dataprep/qa/step/PreparationStep.java]: "ORIGIN" won't always be "/", I think it should be suffixed here too.

File: dataprep-test-api/src/test/resources/features/ExportFormat.feature [hunk 2]
```diff
@@ -1,9 +1,10 @@
-Feature: Test the export format
+Feature: Test OS the export format
 
   @CleanAfter
-  Scenario: Get the export format and verify the S3 export format list
+  Scenario: Get the export format and verify the returned export format
     Given I upload the dataset "/data/6L3C.csv" with name "simpleCSVForExportFormat"
     And I create a preparation with name "simpleExportPrep", based on "simpleCSVForExportFormat" dataset
     When I get the export formats for the preparation "simpleExportPrep"
```
Comment (yacarof) [inline dataprep-test-api/src/test/resources/features/ExportFormat.feature]: I would have preferred a step like : 
```
Then I check that "simpleExportPrep" available export formats are : 
| XLSX | CSV |
```

File: dataprep-test-api/src/test/java/org/talend/dataprep/qa/config/GlobalStep.java [hunk 3]
```diff
@@ -37,36 +43,91 @@
...
+            for (String prepId:listPreparationDeletionPb) {
+                LOGGER.warn("Pb in the deletion of preparation {}.",prepId);
+            }
+            for (String datasetId:listDatasetDeletionPb) {
+                LOGGER.warn("Pb in the deletion of dataset {}.",datasetId);
+            }
+            for (Folder folder:listFolderDeletionPb) {
+                LOGGER.warn("Pb in the deletion of folder {}.",folder.getPath());
+            }
+            LOGGER.warn("The Clean After Step is Ok. All deletion were done.");
```
[the remark this answers is not in the document]
Comment (tlnd-jbrun) [reply dataprep-test-api/src/test/java/org/talend/dataprep/qa/config/GlobalStep.java]: indeed :-)

File: dataprep-test-api/src/test/java/org/talend/dataprep/qa/step/PreparationStep.java (hunk 1 above)
Comment (tlnd-jbrun) [reply dataprep-test-api/src/test/java/org/talend/dataprep/qa/step/PreparationStep.java]: yes, indeed

File: dataprep-test-api/src/test/resources/features/ExportFormat.feature (hunk 2 above)
Comment (tlnd-jbrun) [reply dataprep-test-api/src/test/resources/features/ExportFormat.feature]: Done

**Pull request merged**